tests/draft2019-09/ref.json

```diff
@@ -273,7 +273,7 @@
         "tests": [
             {
                 "description": "valid tree",
-                "data": { 
+                "data": {
```

Comment:
let's do whitespace (and possibly other reformatting) changes in a separate commit/PR.